lib/github_adapter.rb

```diff
@@ -19,11 +16,30 @@ def create_team(name, description)
   end
 
   def invite_to_team(team_id, username)
-    resource = RestClient::Resource.new(
-      "https://api.github.com/teams/#{team_id}/memberships/#{username}",
-      user, password
+    resource = create_resource("https://api.github.com/teams/#{team_id}/memberships/#{username}")
+    response = resource.put({}.to_json, content_type: 'application/json')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.